glean-core/tests/timing_distribution.rs

```diff
@@ -245,6 +252,9 @@ fn the_accumulate_samples_api_correctly_handles_overflowing_values() {
     // Overflowing values are truncated to MAX_SAMPLE_TIME and recorded.
     assert_eq!(snapshot.sum as u64, MAX_SAMPLE_TIME + 6);
 
+    // Check that we got the right number of values.
```

Comment:
Uber-nit: you used "samples" instead of "values" just about everywhere else this comment appears

Reply:
Good call, fixed in [`04fd7db` (#2196)](https://github.com/mozilla/glean/pull/2196/commits/04fd7db311d49b0ee25253d872a1397b87eca0d1)